### ISSUE TITLE
Presolve prints when output_flag=false

### ISSUE THREAD

**Issue by odow** [ERGO-Code/HiGHS#1120]
Reported here: https://github.com/jump-dev/HiGHS.jl/issues/150

MPS file is: [bug.mps.txt](https://github.com/ERGO-Code/HiGHS/files/10551342/bug.mps.txt)

Default solve is:
```julia
julia> HiGHS.HiGHS_jll.highs() do exe
           run(`$(exe) bug.mps`)
       end
Running HiGHS 1.4.0 [date: 1970-01-01, git hash: bcf6c0b22]
Copyright (c) 2022 ERGO-Code under MIT licence terms
LP   bug has 484 rows; 962 cols; 1928 nonzeros
Presolving model
480 rows, 960 cols, 1920 nonzeros
0; Iter: Time   3.982e-07; average =   3.982e-08; Bound =   5.898e-05
100; Iter: Time   4.529e-07; average =   8.113e-08; Bound =    0.000116
200; Iter: Time   2.059e-07; average =   9.361e-08; Bound =    0.000132
300; Iter: Time   2.353e-07; average =   1.078e-07; Bound =   0.0001487
400; Iter: Time   2.264e-07; average =   1.196e-07; Bound =   0.0001616
479 rows, 960 cols, 1916 nonzeros
Presolve : Reductions: rows 479(-5); columns 960(-2); elements 1916(-12)
Solving the presolved LP
Using EKK dual simplex solver - serial
  Iteration        Objective     Infeasibilities num(sum)
          0     0.0000000000e+00 Ph1: 0(0) 0s
        527    -4.7454769715e+05 Pr: 0(0) 0s
        527    -4.7454769715e+05 Pr: 0(0) 0s
Solving the original LP from the solution after postsolve
Model   status      : Optimal
Simplex   iterations: 527
Objective value     : -4.7454769715e+05
HiGHS run time      :          0.01
Process(`/Users/oscar/.julia/artifacts/0bca2cfb4aea96426867c8387de6cd60fa2f39de/bin/highs bug.mps`, ProcessExited(0))
```

When we set an `options` file:
```Julia
shell> cat options
output_flag = false

julia> HiGHS.HiGHS_jll.highs() do exe
           run(`$(exe) bug.mps --options_file=options`)
       end
0; Iter: Time    6.05e-07; average =    6.05e-08; Bound =    8.96e-05
100; Iter: Time   7.803e-07; average =   1.325e-07; Bound =    0.000189
200; Iter: Time   3.439e-07; average =   1.536e-07; Bound =   0.0002161
300; Iter: Time   4.039e-07; average =   1.786e-07; Bound =   0.0002456
400; Iter: Time   3.406e-07; average =   1.948e-07; Bound =   0.0002632
Process(`/Users/oscar/.julia/artifacts/0bca2cfb4aea96426867c8387de6cd60fa2f39de/bin/highs bug.mps --options_file=options`, ProcessExited(0))
```

Running with `--presolve=off` doesn't print:
```julia
julia> HiGHS.HiGHS_jll.highs() do exe
           run(`$(exe) bug.mps --options_file=options --presolve=off`)
       end
Process(`/Users/oscar/.julia/artifacts/0bca2cfb4aea96426867c8387de6cd60fa2f39de/bin/highs bug.mps --options_file=options --presolve=off`, ProcessExited(0))
```

**Comment by jajhall** [comment]
Sorry, this is debugging code that I forgot to remove

**Comment by jajhall** [comment]
Fixed in release 1.4.2
